pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
Hmm

Reply:
I'm on pnpm 8.6.0 as specified in package.json and it insists on this, so I guess it's good for committing